test/errtest.c

```diff
@@ -36,25 +36,37 @@ static int test_print_error_format(void)
     char *out = NULL, *p = NULL;
     int ret = 0, len;
     BIO *bio = NULL;
+    int code = EPERM;
```

Comment:
Why have the code variable, just use EPERM throughout.

Reply:
So someone changes EPERM to some other E-macro in one spot but forgets another...  Why can't one spot be enough?

Reply:
If someone changes it in one place, the test will fail.  This level of indirection makes the code less clear. It cascades, because now you have an auxiliary "code2" variable.  code2 is not a clear name either.  And so on.

You have to at least make code "const int" no?